src/test/java/com/garannodou/role/controller/GoogleAuthRequestTest.java

```diff
@@ -0,0 +1,407 @@
+package com.garannodou.role.controller;
+
+import com.garannodou.role.controller.GoogleAuthRequest.GoogleAuth;
+import com.garannodou.role.controller.GoogleAuthRequest.GoogleBasicProfile;
+import org.junit.jupiter.api.Test;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class GoogleAuthRequestTest {
+
+    /*
+    GoogleBasicProfile field tests
+     */
+    private static Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+
+    @Test
+    public void testNameLengthIsInvalidWhenIsTooShort() {
+        //Given
+        GoogleAuthRequest googleAuthRequest = createValidGoogleAuthRequest();
+        GoogleBasicProfile googleBasicProfile = googleAuthRequest.getGoogleBasicProfile();
+        //When
+        googleBasicProfile.setName("u");
+
+        //Then
+        List<ConstraintViolation<GoogleAuthRequest.GoogleBasicProfile>> constraintViolations =
+                new ArrayList<>(validator.validate(googleBasicProfile));
+
+        assertThat(constraintViolations.size(), is(1));
```

Comment:
Can you put the two asserts together. Like one assert after the other, no have everything that we assert together?
This goes for all tests